question/bank/importquestions/classes/privacy/provider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Privacy Subsystem implementation for qbank_importquestions.
+ *
+ * @package   qbank_importquestions
+ * @category  privacy
+ * @copyright 2021 Catalyst IT Australia Pty Ltd
+ * @author    Marc-Alexandre Ghaly <marc-alexandreghaly@catalyst-ca.net>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace qbank_importquestions\privacy;
+
+class provider implements \core_privacy\local\metadata\null_provider {
```

Comment:
please define the class, something like this:
/**
 * Privacy Subsystem for qbank_viewquestionactions implementing null_provider.
 *
including Catalyt IT and your name as an author